src/lib/AST/ASTVisitor.cpp

```diff
@@ -50,6 +50,14 @@ namespace mrdocs {
 
 namespace {
 
+llvm::StringRef getFullyQualifiedName(NamedDecl const* decl)
+{
+    llvm::SmallVector<char, 128> fullNameBuffer;
+    llvm::raw_svector_ostream fullNameStream(fullNameBuffer);
+    decl->printQualifiedName(fullNameStream);
+    return fullNameStream.str();
```

Comment:
This function returns a `StringRef` constructed from a block scope variable... i.e. a dangling reference. IMO this function should be removed entirely and you should just call `NamedDecl::getQualifiedNameAsString` instead.